src/printer.js

```diff
@@ -1919,6 +1920,10 @@ function printArgumentsList(path, options, print) {
 
   if (groupLastArg) {
```

Comment:
I was thinking of adding this check inside of `groupLastArg`.  We shouldn't group them if the last two have the same type.